spatialdata/_core/models.py

```diff
@@ -645,6 +635,24 @@ def parse(
         region_key: Optional[str] = None,
         instance_key: Optional[str] = None,
     ) -> AnnData:
+        """
+        Parse the :class:`anndata.AnnData` to be compatible with the model.
+
+        Parameters
+        ----------
+        adata
+            :class:`anndata.AnnData`.
```

Comment:
Shouldn't the type of the argument class already be included from the type signature?

```suggestion
```

Reply:
ok, think it'd be nice to also implement a docstring processor likw we have in squidpy